crates/utils/src/rate_limit/mod.rs

```diff
@@ -71,7 +69,8 @@ impl RateLimited {
   {
     // Does not need to be blocking because the RwLock in settings never held across await points,
     // and the operation here locks only long enough to clone
-    let rate_limit: RateLimitConfig = Settings::get().rate_limit.unwrap_or_default();
+    // let rate_limit: RateLimitConfig = Settings::get().rate_limit.unwrap_or_default();
+    let rate_limit = self.rate_limit_config;
```

Comment:
Needed to initialize every struct here with the rate_limit_config, since this is no longer available as the singleton.